test/harry/main/org/apache/cassandra/harry/model/BytesPartitionState.java

```diff
@@ -53,50 +66,118 @@ public class BytesPartitionState
     public final Clustering<ByteBuffer> key;
     private final Token token;
     private final PartitionState state;
+    private final Map<Symbol, MultiCell> staticMultiCell;
+    @Nullable
+    private final Map<Long, Map<Symbol, MultiCell>> rowMultiCell;
```

Comment:
base harry supports cell resolution for non-mutli cell columns but doesn't support multi cell.  Spoke with Alex and said he didn't need this PR to push this logic into base harry, so leaving here for the moment...

I do think this "could" be unified better... having a concept of `Cell` in the API would let me push this logic to the executor layer rather than have this lagic also handle execution... that is a larger refactor and something to think through.